js/gated-content/src/views/SchedulePage.vue

```diff
@@ -148,52 +147,16 @@ export default {
       this.endDate = new Date(this.startDate);
       this.endDate.setTime(this.startDate.getTime() + this.oneWeek - 1);
 
-      const params = {
-        filter: {
-          dateFilterStart: {
-            condition: {
-              path: 'date.value',
-              operator: '>=',
-              value: new Date(
-                this.startDate.getFullYear(),
-                this.startDate.getMonth(),
-                this.startDate.getDate(),
-                0,
-                0,
-                0,
-              ),
-            },
-          },
-          dateFilterEnd: {
-            condition: {
-              path: 'date.value',
-              operator: '<',
-              value: new Date(
-                this.endDate.getFullYear(),
-                this.endDate.getMonth(),
-                this.endDate.getDate(),
-                23,
-                59,
-                59,
-              ),
-            },
-          },
-          status: 1,
+      client({
+        url: '/api/events',
```

Comment:
I think it is better to use `/virtualy/api/events` in order not to conflict with API's from another modules.